runtimes/aspnetcore-1.0/structural_tests/1.0.9/aspnet.yaml

```diff
@@ -0,0 +1,20 @@
+schemaVersion: "1.0.0"
+
+commandTests:
+  - name: 'dotnet installation'
+    command: ['which', 'dotnet']
+    expectedOutput: ['/usr/bin/dotnet\n']
+  - name: 'dotnet version'
+    command: ['dotnet', '--version']
+    expectedOutput: ['.*Version  : 1\.0\.1.*']
```

Comment:
Should this check for 1.0.9?

Reply:
No, 1.0.1 is the version of the `dotnet` binary itself.

Reply:
The versioning in .NET Core is quite confusing.